Gopkg.toml

```diff
@@ -13,14 +13,21 @@ required = [
     "sigs.k8s.io/controller-runtime/pkg/runtime/signals",
     "sigs.k8s.io/controller-runtime/pkg/source",
     "sigs.k8s.io/testing_frameworks/integration", # for integration testing
-	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1",
+  "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1",
+  "github.com/tensorflow/tensorflow/tensorflow/go", # for tensorflow protos
     ]
 
 [prune]
   go-tests = true
   unused-packages = true
   non-go = true
 
+[[prune.project]]
+  name = "github.com/tensorflow/tensorflow"
+  non-go = false
```

Comment:
Isn't this true?

Reply:
I ran `dep ensure` with&without certain fields in [[prune.project]], and with&without [[prune.project]]. `dep ensure` seems to ignore [[prune.project]] completely and cares only about the required array. Removing the [[prune.project]] section